packages/perseus-editor/src/components/scrollless-number-text-field.tsx

```diff
@@ -0,0 +1,45 @@
+import {TextField} from "@khanacademy/wonder-blocks-form";
+import * as React from "react";
+
+import type {PropsFor} from "@khanacademy/wonder-blocks-core";
+
+/**
+ * This is a custom text field of type="number" for use in Perseus Editors.
+ *
+ * This makes it so that the text field's input number updates on scroll
+ * without scrolling the page.
+ *
+ * NOTE 1: Native HTML number inputs do not update the number value on scroll,
+ * they only scroll the page. For some reason, inputs in React do NOT work
```

Comment:
The reason this happens is explained here: https://stackoverflow.com/a/68266494

Might be worth linking to that answer so future maintainers know what's up.